snap/squashfs/squashfs.go

```diff
@@ -96,7 +96,7 @@ var runCommand = func(args ...string) error {
 	return err
 }
 
-func (s *Snap) unpack(src, dstDir string) error {
+func (s *Snap) Unpack(src, dstDir string) error {
```

Comment:
Yes! That is the goal, I am happy to do this in this branch or in a followup branch, whatever is more convenient for you to review.